src/NzbDrone.Core.Test/ParserTests/ParsingServiceTests/MapFixture.cs

```diff
@@ -20,14 +20,15 @@ public class MapFixture : TestBase<ParsingService>
         private Series _series;
         private List<Episode> _episodes;
         private ParsedEpisodeInfo _parsedEpisodeInfo;
+        private ParsedEpisodeInfo _umlautInfo;
         private SingleEpisodeSearchCriteria _singleEpisodeSearchCriteria;
 
         [SetUp]
         public void Setup()
         {
             _series = Builder<Series>.CreateNew()
-                .With(s => s.Title = "30 Stone")
-                .With(s => s.CleanTitle = "stone")
+                .With(s => s.Title = "Die Pfefferkörner")
```

Comment:
This is not the correct place to to test this change. Tests for this functionality should be in `ParserFixture`, same as is done with `should_remove_accents_from_title`

Reply:
Should I completely revert MapFixture then?
I just put them there because Radarr also had a Test for umlauts there.

Reply:
Yeah I think so.